web/client/src/components/info/ModuleInfo.jsx

```diff
@@ -42,22 +44,45 @@ export default function ModuleInfo(props) {
         <HelpOutlineIcon />
       </IconButton>
 
-      <Dialog onClose={handleClose} open={open}>
+      <Dialog fullWidth maxWidth="sm" onClose={handleClose} open={open}>
```

Comment:
Could you change the dialog and text to appear bigger in order to accommodate to users with eyesight problems?

Reply:
Done!